book/src/building-from-source.md

```diff
@@ -35,10 +35,18 @@ RUSTFLAGS="-C target-feature=-crt-static"
 2. Compile from source:
 
    ```sh
+   # Reproducible
    cargo install --path helix-term --locked
    ```
+   ```sh
+   # Optimized
+   cargo +stable install \
+      --profile opt \
+      --config 'build.rustflags="-C target-cpu=native"' \
+      --path helix-term
```

Comment:
Looks like we also need `--locked` here. The install command doesn't strictly respect the lockfile without `--locked` specified and that can cause install failures if dependencies update their MSRV ahead of us.